externs/ima.js

```diff
@@ -41,6 +41,7 @@ google.ima.AdsLoader = class {
 
 /**
  * @implements {EventTarget}
+ * @exportDoc
```

Comment:
I don't see why this should be part of the generated API docs if it's an external library that's not part of Shaka Player.

Reply:
Same as above, this has to be exported to the TypeScript definitions one way or the other so the shaka-player typings can refer to this type.